src/inc/dacprivate.h

```diff
@@ -20,6 +20,14 @@
 #include <msodw.h>
 #endif // FEATURE_PAL
 
+#if !defined(MSLAYOUT)
```

Comment:
How about adding new `src/inc/dacstructure.h` containing comments and MSLAYOUT definition? or `daclayout.h`.